ChangeLog

```diff
@@ -1,3 +1,7 @@
+2021-01-06  Conrad Sanderson  <conrad.sanderson.nospam@nogmail.com>
```

Comment:
Appreciate the effort but not a valid email address.

Reply:
fixed